Documentation/Makefile

```diff
@@ -88,7 +88,7 @@ $(HELM_VALUES): FORCE
 	$(QUIET)printf '..\n  %s\n\n%s\n' "AUTO-GENERATED. Please DO NOT edit manually." "$$(cat $@)" > $@
 	$(QUIET)$(RM) -- $(TMP_FILE_1) $(TMP_FILE_2)
 
-epub latex html: builder-image copy-api update-helm-values
+epub latex html: clean builder-image copy-api update-helm-values
```

Comment:
Yes, this target is not idempotent because of the `copy-api` step. The 1st time it's invoked,  it copies  `../api` to `_api` and the 2nd time, as the `_api` directory already exists, it gets copied to `_api/api`. I guess a better fix would be to `rm -rf _api` at the `copy-api` target.

Reply:
Even better if we could copy correctly. I think we could pass `-T` to `cp` to avoid copying into `_api/`?

```
copy-api:
	@$(ECHO_GEN)_api
	$(QUIET)cp -T -r ../api _api
```